src/main/java/net/sf/mzmine/modules/tools/isotopepatternpreview/customparameters/IsotopePatternPreviewCustomParameters.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright 2006-2018 The MZmine 2 Development Team
+ * 
+ * This file is part of MZmine 2.
+ * 
+ * MZmine 2 is free software; you can redistribute it and/or modify it under the terms of the GNU
+ * General Public License as published by the Free Software Foundation; either version 2 of the
+ * License, or (at your option) any later version.
+ * 
+ * MZmine 2 is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * General Public License for more details.
+ * 
+ * You should have received a copy of the GNU General Public License along with MZmine 2; if not,
+ * write to the Free Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301
+ * USA
+ */
+
+
+package net.sf.mzmine.modules.tools.isotopepatternpreview.customparameters;
+
+import java.text.DecimalFormat;
+import net.sf.mzmine.main.MZmineCore;
+import net.sf.mzmine.parameters.Parameter;
+import net.sf.mzmine.parameters.impl.SimpleParameterSet;
+import net.sf.mzmine.parameters.parametertypes.DoubleParameter;
+import net.sf.mzmine.parameters.parametertypes.PercentParameter;
+
+/**
+ * 
+ * @author Steffen Heuckeroth steffen.heuckeroth@gmx.de / s_heuc03@uni-muenster.de
+ *
+ */
+public class IsotopePatternPreviewCustomParameters  extends SimpleParameterSet{
+
+  public static final PercentParameter minAbundance = new PercentParameter("Minimum abundance",
+      "The minimum abundance (%) of Isotopes", 0.01);
+  
+  public static final DoubleParameter mergeWidth = new DoubleParameter("Merge width(Da)",
+      "This will be used to merge isotope compositions in the calculated isotope pattern if they overlap.",
+      MZmineCore.getConfiguration().getMZFormat(), 0.0005, 0.0d, 10.0d);
+  
+  public static final DoubleParameter minPatternIntensity =
+      new DoubleParameter("Min. pattern intensity",
+          "The minimum normalized intensity of a peak in the final calculated isotope pattern.\n"
+              + "Min = 0.0, Max = 0.99...",
+          new DecimalFormat("0.####"), 0.01, 0.0d, 0.99999);

```

Comment:
Why not use MZmineCore.getConfiguration().getIntensityFormat() ?

Reply:
It will say 1.0E-2 after closing the dialog instead of 0.01. The y-axis of the plot is normalized, so the decimal format makes it more intuitive, I think. I will change it to MZmineCore.getConfiguration().getIntensityFormat() if you want, though.

Reply:
No that's fine, I didn't realize that this is relative intensity. Perhaps you should indicate that in the name ("Min. relative pattern intensity")